File: Haskell/beginner/1837.hs
-- https://judge.beecrowd.com/pt/problems/view/1837
-- Prefácio

-- Main
main :: IO ()
main = do
    usrInput <- getLine
    let [dividend, divider] = map read (words usrInput) :: [Int]

    let rest = dividend `mod` divider
    let rest' = if rest < 0 then rest + abs divider else rest

    let quotient = (dividend - rest') `div` divider

    putStrLn $ show quotient ++ " " ++ show rest'


-- By: Fernando Serra
-- https://github.com/fernandosserra

File: Python/beginner/1837.py
# https://judge.beecrowd.com/pt/problems/view/1837
# Prefácio

dividend, divider = map(int, input().split())

remainder = dividend % divider

if remainder < 0:
    remainder += abs(divider)

quotient = (dividend - remainder) // divider
print(quotient, remainder)

# By: Fernando Serra
# https://github.com/fernandosserra